grails-app/domain/org/pih/warehouse/order/Order.groovy

```diff
@@ -398,7 +398,7 @@ class Order implements Serializable {
      * @return true if order has a regular invoice; false otherwise
      */
     Boolean getHasRegularInvoice() {
-        return invoices.any { it.invoiceType == null || it.invoiceType?.code == InvoiceTypeCode.INVOICE }
+        return invoices.any { it.invoiceType == null || it.invoiceType?.code == InvoiceTypeCode.PURCHASE_INVOICE }
```

Comment:
Should it be only purchase invoice or other types of invoices too?
I think it should be INVOICE, PURCHASE_INVOICE and SALES_INVOICE
@awalkowiak what do you think?

Reply:
I agree, previously these three types had code `INVOICE`, now after splitting this should be covered here too

Reply:
Done